client/src/curriculum/module.scss

```diff
@@ -21,7 +29,7 @@
         }
       }
 
-      p {
+      p.module-topic {
```

Comment:
Nit: Do we need to be this explicit?

```suggestion
      .module-topic {
```

Reply:
No, but we can. That's a discussion for having CSS guidelines in the future.